tests/test-utils.php

```diff
@@ -143,5 +143,13 @@ public function testAssocArgsToString() {
 		) ) );
 	}
 
+	public function testMaybePreviewEnv() {
+		if ( 0 === strncasecmp( 'WIN', PHP_OS, 3 ) ) {
```

Comment:
Maybe a test arg like `is_windows( $test = null )` instead - an env var is a bit "dangerous"?!